### PR TITLE
feature: timezone support for graylog

### PR DESCRIPTION
**Pull request by crcro**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

feature request from community site

before:
screen shot 2017-11-26 at 4 40 06 pm

after:
screen shot 2017-11-26 at 4 39 45 pm

### CODE REVIEW

**Comment by murrant** [comment]
Not sure I like the way this is fixed.   I think they should just be displayed in the timezone of the browser accessing them.

**Comment by murrant** [comment]
http://www.jquery-bootgrid.com/documentation#converters

**Comment by murrant** [comment]
If you don't want to force it on the user, you should add a timezone setting to the user settings.  With browser timezone as the default.

That is a much larger scope.  Up to you.

**Comment by crcro** [comment]
@murrant: so I made it like:

if no timezone is set in config, print default user timezone using moment.js.
if timezone is set then display using php date functions.

is this better?

**Comment by laf** [comment]
Any reason why you've removed my name from the copyright?

**Comment by crcro** [comment]
By mistake, copy/paste the entire copyright section, restored it. Sorry

**Comment by murrant** [comment]
Yeah, seems a like a good compromise for now.  Thanks.

File: html/includes/table/graylog.inc.php [hunk 1]
```diff
@@ -1,7 +1,6 @@
 <?php
-
 /*
- * LibreNMS
+ * Librenms
```
Comment (laf) [inline html/includes/table/graylog.inc.php]: Really petty but this should still be `LibreNMS`

**Pull request merged**